azure-pipelines.yml

```diff
@@ -107,25 +107,16 @@ jobs:
       packDirectory: $(Build.ArtifactStagingDirectory)/deployables
 
   - powershell: >-
-      $testdir="bin\Nerdbank.Streams.Tests\$(BuildConfiguration)\net461"
-
-      & "$HOME\.nuget\packages\OpenCover\4.6.519\tools\OpenCover.Console.exe"
-      -register:user
-      -target:"$HOME\.nuget\packages\xunit.runner.console\2.3.1\tools\net452\xunit.console.x86.exe"
-      -targetargs:"$testdir\Nerdbank.Streams.Tests.dll -noshadow -parallel all -notrait SkipInCodeCoverage=true -html $testdir\testresults.html -xml $testdir\testresults.xml -appveyor -nologo"
-      -returntargetcode
-      -excludebyattribute:*.ExcludeFromCodeCoverage*
-      -excludebyfile:*\*Designer.cs
-      -filter:"+[Nerdbank.Streams]*"
-      -hideskipped:All
-      -output:$testdir\code_coverage.xml
+      $testdir="src\Nerdbank.Streams.Tests"
```

Comment:
nit: At this point I think it makes more sense to just remove this variable declaration and inline the value in the two used places.